src/gpg/key/rev.py

```diff
@@ -0,0 +1,17 @@
+import gpg
+import sys
+
+def revoke(revcert_path):
+    c = gpg.Context()
+    with open(revcert_path) as revcert_file:
+        c.op_import(revcert_file)
+        result = c.op_import_result()
+
+        if result is None:
+            raise Exception("Could not import revocation certificate")
+        else:
+            print("Successfully imported revocation certifiate")
+
```

Comment:
The user is asking to revoke the existing key. So, this message in my opinion, does not make much sense to the user. The same for the error message.

I would suggest that the python script just return an exit code (0 when successful, anything else if there is an error). The bash script can check this exit code and print a message that is appropriate.

The messages can be like this: "Key revoked successfully." and "Key revocation failed."

Reply:
Ok returning a status code seems fine to me. gpg does the same we only issue an error message in  case of a gpgme exception/some other error occurs.